server/app/main.py

```
@@ -0,0 +1,45 @@
+from fastapi import FastAPI, Request, HTTPException
+from fastapi.responses import JSONResponse
```

Ruff

`fastapi.responses.JSONResponse` imported but unused

Remove unused import: `fastapi.responses.JSONResponse`

(F401)